src/modules/vtol_att_control/standard.cpp

```diff
@@ -149,8 +149,8 @@ void Standard::update_vtol_state()
 				const Vector3f vel = R_to_body * Vector3f(_local_pos->vx, _local_pos->vy, _local_pos->vz);
 				exit_backtransition_speed_condition = vel(0) < _params->mpc_xy_cruise;
 
-			} else if (PX4_ISFINITE(_airspeed_validated->calibrated_airspeed_m_s)) {
-				exit_backtransition_speed_condition = _airspeed_validated->calibrated_airspeed_m_s < _params->mpc_xy_cruise;
+			} else if (PX4_ISFINITE(*_airspeed_smooth)) {
+				exit_backtransition_speed_condition = *_airspeed_smooth < _params->mpc_xy_cruise;
```

Comment:
Have we done a risk analysis of this? What if something is wrong with the smoothing, so it back-transitions much too soon or much too late / not at all?

Reply:
Time condition with `VT_B_TRANS_DUR` is a safeguard for late exit.

Since an LPFed signal will have a delay compared to raw measurements, early exit is less likely than with non-lpfed signal. In this case, top motors would immediately start, with a similar effect to quad-chute.